contrib/untar.c

```diff
@@ -133,18 +144,19 @@ verify_checksum(const char *p)
 static void
 untar(FILE *a, const char *path)
 {
-	char buff[512];
+#define BUF_SIZE 512
+	char buff[BUF_SIZE];
 	FILE *f = NULL;
 	size_t bytes_read;
```

Comment:
`bytes_read` here could be `unsigned` or even just `int` since you know it will always be less than `BUF_SIZE`.  A plain `int` type will also avoid confusing warnings from things like `bytes_read < BUF_SIZE` since BUF_SIZE itself is an int.